features/redesigned_searches.feature

```diff
@@ -147,20 +147,6 @@ Feature: Search - redesign
     And I should see an image link to "USAJobs.gov" with url for "https://www.usajobs.gov/"
     And I should see a link to "More federal job openings on USAJobs.gov" with url for "https://www.usajobs.gov/Search/Results?hp=public"
 
-    When I am on en.agency.gov's search page
-    And I fill in "Enter your search term" with "blablah jobs"
-    And I press "Search"
-    Then I should see an image link to "USAJobs.gov" with url for "https://www.usajobs.gov/"
-    And I should see "No job openings in your region match your search"
-    And I should see a link to "More federal job openings on USAJobs.gov" with url for "https://www.usajobs.gov/Search/Results?hp=public"
-
-    When I am on es.agency.gov's search page
-    And I fill in "Ingrese su búsqueda" with "blablah trabajo"
-    And I press "Buscar"
-    Then I should see an image link to "USAJobs.gov" with url for "https://www.usajobs.gov/"
-    And I should see "Ninguna oferta de trabajo en su región coincide con su búsqueda"
-    And I should see a link to "Más trabajos en el gobierno federal en USAJobs.gov" with url for "https://www.usajobs.gov/Search/Results?hp=public"
-
```

Comment:
removed test for unavailable jobs as backend doesnt tell when there are no jobs, wip at SRCH-4577